tests/integration/test_author_api.py

```diff
@@ -45,28 +45,38 @@ def test_api_authors_citations(app):
             'properties': {
                 'citee': {
                     'properties': {
+                        'id': {'type': 'integer'},
                         'record': {
                             'properties': {'$ref': {'type': 'string'}},
                             'type': 'object'
                         }
                     },
                     'type': 'object'
                 },
-                'citer': {
-                    'properties': {
-                        'record': {
-                            'properties': {'$ref': {'type': 'string'}},
-                            'type': 'object'
-                        }
+                'citers': {
+                    'items': {
+                        'properties': {
+                            'citer': {
+                                'properties': {
+                                    'id': {'type': 'integer'},
+                                    'record': {
+                                        'properties': {'$ref': {'type': 'string'}},
+                                        'type': 'object'
+                                    }
+                                },
+                                'type': 'object'
+                            },
+                            'date': {
+                                'format': 'date',
+                                'type': 'string'
+                            },
+                            'published_paper': {'type': 'boolean'},
+                            'self_citation': {'type': 'boolean'}
+                        },
+                        'type': 'object'
                     },
-                    'type': 'object'
-                },
-                'date': {
-                    'format': 'date',
-                    'type': 'string'
-                },
-                'published_paper': {'type': 'boolean'},
-                'self_citation': {'type': 'boolean'}
+                    'type': 'array'
+                }
             },
             'type': 'object'
```

Comment:
@jacenkow note that there was a reason the schema had the types at the bottom: I expect these schemas to be moved in `inspirehep/modules/records/jsonschemas/api/authors`, where `scripts/prettifyjson` will run on them. So if they are already in a good order here the diff will be very clean when that happens (just replace the `"` with `'`).